tap_github/schemas/releases.json

```diff
@@ -36,11 +178,68 @@
       "type": ["null", "object"],
       "additionalProperties": false,
       "properties": {
+        "name": {
+          "type": ["null", "string"]
```

Comment:
We used author field on multiple places, can we move this in common?

Reply:
Author was similar to user, hence used it